source/likely.js

```diff
@@ -1,6 +1,18 @@
 'use strict';
 
 var likely = require('./index.js');
+var history = require('./history');
+var utils = require('./utils');
 
 window.likely = likely;
-window.addEventListener('load', likely.initiate);
+window.addEventListener('load', function () {
+    if (likely.defaults.listenChangesUrl) {
```

Comment:
Single Page Application. Usually it has complex logic and listens url changes by itself. It such case a user may want to run the process manually.